src/applications/personalization/rated-disabilities/components/TotalRatedDisabilities.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import PercetnageCalloutBox from '../components/PercentageCalloutBox';
+
+const TotalRatedDisabilities = props => {
+  const { loading, totalDisabilityRating, error } = props;
+  let content;
+  // If the data from the parent is loading, show a loading indicator
+  // If there is an error, display an error message,
+  // If there is no rating, display a no rating message
+  // If there is a rating, display the rating and content
+  if (loading) {
+    content = (
+      <LoadingIndicator message="Loading your total disability rating..." />
+    );
+  } else if (error) {
+    const message = (
+      <span>
+        <p>
+          We're sorry. An error occurred when accessing your disability rating
+          information.
+        </p>
+        <h4>What you can do</h4>
+        <p>
+          Sign out of VA.gov, then log back in to try this page again. If the
+          error continues, please call the VA.gov Help Desk at 1-855-574-7286
+          (TTY:1-800-829-4833). We're here Monday-Friday, 8:00 a.m. - 8:00 p.m.
+          (ET).
+        </p>
+      </span>
+    );
+    content = (
+      <AlertBox
+        headline="Total disabilities error"
+        content={message}
+        status="error"
+        isVisible
+      />
+    );
+  } else if (!totalDisabilityRating) {
+    const message = (
+      <span>
+        <p>We don't have a disability rating for you in our system.</p>
+        <h4>Want to add disabilities?</h4>
+        <p>
+          If you believe you have disabilities related to your military service,
+          you can submit a claim to update your disability rating.
+        </p>
+        <a href="#" className="usa-link">
+          Submit a new claim
+        </a>
+      </span>
+    );
+    content = (
+      <AlertBox
+        headline="No disability rating to show"
+        content={message}
+        status="info"
+        isVisible
+      />
+    );
+  } else {
+    content = (
+      <span>
+        <div className="total-rated-disabilities">
+          <div className="vads-l-row">
+            <div className="vads-l-col--12">
+              <h1>Your disability rating</h1>
+            </div>
+          </div>
+          <div className="vads-l-row">
+            <div className="vads-l-col--12 medium-screen:vads-l-col--6 small-desktop-screen:vads-l-col--9 vads-u-padding-right--1p5">
+              <p>
+                <strong>
+                  Your combined disability rating is {totalDisabilityRating}%
+                </strong>
+                . This rating determines the amount of compensation you'll get.
```

Comment:
The mockup suggests this should read: This rating determines the amount of _disability_ compensation you'll get.

Reply:
@micahchiang Yeah, this is all dummy content. Jim said much of the content will change before we are ready to go live.

Reply:
@micahchiang Just pushed 74a3d89 to fix this